pkgs/by-name/an/anytype/tantivy-go/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "tantivy-go";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "anyproto";
+    repo = "tantivy-go";
+    tag = "v${version}";
+    hash = "sha256-IlGtyTjOAvmrbgmvy4NelTOgOWopxNta3INq2QcMEqY=";
+  };
+
+  cargoHash = "sha256-2fprQHpepF6n+aovq3U4JQue7xmTDUZMnzDLEgc7ock=";
```

Comment:
might have a cargoHash mismatch

Reply:
I think what's happening is the problem described in #377996 and other similar recent PRs. I've enabled `useFetchCargoVendor` which seems to be the recommended way to solve this.